recipes/tvm/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "tvm" %}
```

Comment:
Should the name have a `python` or `py` in it because this is the python wrapper?

Reply:
Should we slipt this recibe as ``tvm-libs`` and ``py-tvm`` then?